save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/storage/AbstractFileBasedStorage.kt

```diff
@@ -55,10 +55,6 @@ abstract class AbstractFileBasedStorage<K>(
         val contentPath = buildPathToContent(key)
         return Mono.fromCallable {
             contentPath.deleteIfExists()
-        }.doOnNext {
-            if (it) {
-                contentPath.parent.deleteDirectoriesTill(rootDir)
```

Comment:
why do remove cleanup of directories ?

Reply:
`AbstractFileBasedStorage` is a common class -- it does affect all storages. Please rollback.

Also, even in case of `AvatarStorage`, this cleanup is required to delete folders when all avatars are removed

Reply:
Rollbacked it